package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mattrglobal/bbs-signatures",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
Shouldn't this be:

```json
  "version": "2.0.0",
```

Because of BREAKING CHANGE

Reply:
It seems this is the precedent: https://github.com/mattrglobal/bbs-signatures/blob/5bd9ae00ac6f9ebaf43af2295fd5ad4188e1e9ee/CHANGELOG.md#features-1

It is a breaking change in the engine but the api is the same

Reply:
Updated to version 2